arch/arm/src/Makefile

```diff
@@ -138,6 +143,13 @@ $(KBIN): $(OBJS)
 board$(DELIM)libboard$(LIBEXT):
 	$(Q) $(MAKE) -C board libboard$(LIBEXT) EXTRAFLAGS="$(EXTRAFLAGS)"
 
+ifeq ($(CONFIG_LTO_FULL),y)
```

Comment:
why do we need `ifeq ($(CONFIG_LTO_FULL),y)` here?

Reply:
Done